src/FlappyTinybirdScene.js

```diff
@@ -0,0 +1,105 @@
+import Phaser from "phaser";
+import { send_score } from "./tinybird";
+import { v4 as uuidv4 } from "uuid";
+
+export default class FlappyTinybirdScene extends Phaser.Scene {
+  scoreText;
+  score = 0;
+  pipes;
+  bird;
+  timer;
+  session = {};
+
+  constructor() {
+    super({ key: "FlappyTinybirdScene" });
+  }
+
+  preload() {
+    this.session.id = uuidv4();
+    this.session.name = "test";
+    this.session.email = "test@test.test";
+
+    this.load.image("bird", "assets/bird.png");
+    this.load.image("pipe", "assets/pipe.png");
+    this.canvas = this.sys.game.canvas;
+  }
+
+  create() {
+    this.bird = this.physics.add.sprite(100, 245, "bird");
+
+    const spaceKey = this.input.keyboard.addKey(
+      Phaser.Input.Keyboard.KeyCodes.SPACE
+    );
+
+    spaceKey.on("down", (key, event) => {
```

Comment:
We should make this mobile friendly. I added `this.input.onDown.add(this.jump, this);` in my old version to get this to work on mobile.

Reply:
👍 added